monitoring/Tenant_Monitoring.md

```diff
@@ -0,0 +1,58 @@
+# Deploying Monitoring for Tenants
+
+## Overview
+
+If you are the provider administrator in a multi-tenant environment, you can 
+provide monitoring capabilities for the tenant administrators in your 
+environment. The tenant administrator can view CAS and SAS job metric 
+information for only their tenant. The provider administrator can view 
+information for all tenants.
+
+Monitoring components are deployed individually for each tenant, so you 
+can choose which tenants can have monitoring capability. The deployment 
+process deploys tenant-specific instances of Prometheus and Grafana that 
+process and display information only from the tenant. 
```

Comment:
Should we go into more detail on the fact that each tenant deployment deploys its own instance of both Prometheus and Grafana. Resource requirements are smaller since Prometheus is not collecting information from the full cluster. The new Grafana instance will have its own, separate user definitions and only 3 dashboards.

Reply:
I added some more information to provide more detail and clarification